environments/stfc-base/inventory/group_vars/all/user_cluster_k8s_app_configs.yml

```diff
@@ -1,5 +1,14 @@
 azimuth_capi_operator_app_templates_jupyterhub_default_values:
   jupyterhub:
+    prePuller:
+      # Disabled because of large GPU images
+      # See https://z2jh.jupyter.org/en/latest/resources/reference.html#prepuller-pullprofilelistimages for alternatives should we wish to renable
+      # I.e. we could add non-gpu images to extraImages
+      hook:
+        enabled: false 
+      continuous:
+        enabled: false
```

Comment:
Leave the hook disabled but continuous pre-pull on
That way deploying the config doesn't block, but it can pull these large images in the background (through our dockerhub cache) so a user doesn't have to wait 20 minutes from selecting their env